deploy/kubernetes/base/node_windows/node.yaml

```diff
@@ -43,6 +43,7 @@ spec:
           # Don't change base image without changing pdImagePlaceholder in
           # test/k8s-integration/main.go
           image: gke.gcr.io/gcp-compute-persistent-disk-csi-driver-win
+          imagePullPolicy: Always
```

Comment:
We usually only set to always for dev environments where we're rebuilding the same image, but in general, we leave it unset (default)

Reply:
updated.